app/assets/javascripts/evercookie/evercookie.js.erb

```diff
@@ -354,10 +356,14 @@ var <%= Evercookie.js_class %> = (function (window) {
                         }
                     };
                 }
-                img.src = "<%= Evercookie.get_png_path %>?name=" + name;
+                img.src = this.full_url("<%= Evercookie.get_png_path %>", {name: name});
             }
         };
 
+        this.full_url = function(path, params) {
+            return this.getHost() + path + '?' + $.param(params)
```

Comment:
@csigg just realized this, I don't think you can use $ since we can't assume jQuery will be on the page